contributors.yml

```
@@ -50,7 +50,8 @@ authors:
     email: jeremy.cohen@imperial.ac.uk
     acknowledgements: |
       JC acknowledges support from the UK Engineering and Physical Sciences
-      Research Council (UKRI-EPSRC) under grants EP/R025460/1 and EP/W035731/1.
+      Research Council (UKRI-EPSRC) under grants EP/R025460/1, EP/W035731/1 and
```

Not a fan of the Oxford comma? :smile: 

```suggestion
      Research Council (UKRI-EPSRC) under grants EP/R025460/1, EP/W035731/1, and
```

---

Good point - I don't think I'm very consistent with these things, sometimes it seems to work but I thought here that it didn't! 😄